phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -484,6 +484,16 @@ private void openConnection() throws SQLException {
         }
     }
 
+    // Close the HBase Connection
+    private void closeConnection() throws IOException {
+        if (connection != null) {
+            LOGGER.info("HConnection closed. Stacktrace for informational purposes: "
+                + connection + " " + LogUtil.getCallerStackTrace());
+            connection.close();
```

Comment:
Put the log line after the call. If `connection.close()` throws then the log line will be misleading.